naughty/fedora-30/118-selinux-rpcbind-name_bind

```diff
@@ -1 +1 @@
-testlib.Error: audit: type=1400 audit(*): avc:  denied  { name_bind } * comm="rpcbind"
+* audit: type=1400 audit(*): avc:  denied  { name_bind } * comm="rpcbind"
```

Comment:
heh, this cannot be like this, there is no space before audit. And also it is not "consistent pattern" :)

Reply:
Argh, sorry -- `git grep audit:` is empty now. Wash, rinse, repeat! (Pushed)